apps/qa/src/pages/ingest/ingest.py

```diff
@@ -0,0 +1,20 @@
+def ingest():
+    import streamlit as st
+    from .helpers import archive_raw_data
+    from pathlib import Path
+
+    st.title("Ingest Dataset")
+
+    dataset_name = st.text_input("Dataset Name")
+    version = st.text_input("Version")
+    uploaded_file = st.file_uploader("Choose a file")
+    s3_path = Path("inbox") / dataset_name / version
+
+    if st.button("Ingest"):
```

Comment:
This does work, but I think I would prefer setting a variable equal to the call to `st.button`. I think it just "flows" a little better. "if button" is a slightly confusing boolean value. Something like `result = st.button()` or `ingest_button_pressed = st...` makes it clear what value is being returned by the input, and helps separate out the __visual__ code of calling `st` components and the more logical/control flow code of checking its boolean value